spring-reactive-modules/spring-reactive-client-2/src/main/java/com/baeldung/fluxmapandondonext/DoOnNext.java

```diff
@@ -0,0 +1,10 @@
+package com.baeldung.fluxmapandondonext;
+
+import reactor.core.publisher.Flux;
+
+public class DoOnNext {
+
+    public Flux<Integer> number_service_range() {
+        return Flux.just(10, 11, 12, 13, 14);
+    }
+}
```

Comment:
Do we need this and `FluxMap.java`?

Reply:
Hi @ashleyfrieze, both DoOnNext.java and FluxMap.java removed.